### PR TITLE
jax_numpy_degrees implimentation

### PR DESCRIPTION
**Pull request by ndurumo254**
#7477 degrees

### CODE REVIEW

**Comment by ndurumo254** [comment]
#7477

**Comment by ndurumo254** [comment]
#7492

**Comment by ndurumo254** [comment]
hello @fspyridakos.
am getting this error when i ran it from my local machine. how should i go about it.

E   ivy.exceptions.IvyError: tensorflow: function_supported_devices_and_dtypes: tensorflow: function_supported_dtypes: tensorflow: function_supported_dtypes: invalid literal for int() with base 10: '0-dev20221117'

**Comment by ndurumo254** [comment]
> hello @fspyridakos. am getting this error when i ran it from my local machine. how should i go about it.
> 
> E ivy.exceptions.IvyError: tensorflow: function_supported_devices_and_dtypes: tensorflow: function_supported_dtypes: tensorflow: function_supported_dtypes: invalid literal for int() with base 10: '0-dev20221117'

am also facing lint errors from other peoples contributions. is it advisable for me to do any changes on those files?

File: ivy/functional/frontends/jax/numpy/name_space_functions.py [hunk 1]
```diff
@@ -558,6 +556,11 @@ def log10(x):
     return ivy.log10(x)
 
 
+@to_ivy_arrays_and_back
+def degrees(x):
+    return ivy.degrees(x)
```
Comment (fspyridakos) [inline ivy/functional/frontends/jax/numpy/name_space_functions.py:561]: There is no `ivy.degrees` function, please use `ivy.rad2deg`

**Review by fspyridakos — CHANGES_REQUESTED** [review]
I've requested a change in a line of code that should make your function run. Otherwise, this PR includes changes to many files you shouldn't be changing, like `test_trigonometric_functions` and `test_array_contents`, please make sure you only have `name_space_functions` and `test_jax_numpy_namespace_functions` showing up in the "Files changed" tab. Additonally it seems you have added extra indentation in many function definitions, please revert these changes too.

**Comment by fspyridakos** [comment]
> hello @fspyridakos. am getting this error when i ran it from my local machine. how should i go about it.
> 
> E ivy.exceptions.IvyError: tensorflow: function_supported_devices_and_dtypes: tensorflow: function_supported_dtypes: tensorflow: function_supported_dtypes: invalid literal for int() with base 10: '0-dev20221117'

Where do you get this error? When running the test? Also about the lint errors, you do not have to fix any lint errors in any files apart from the ones you're modifying. If you would like automatic lint checking, please check out the "pre-commit" section in our setting up giude: https://lets-unify.ai/ivy/contributing/setting_up.html#pre-commit

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py [hunk 2]
```diff
@@ -3060,6 +3060,37 @@ def test_jax_numpy_log10(
...
 
 
+# degrees
+@handle_frontend_test(
+    fn_tree="jax.numpy.deg2rad",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+)
+def test_jax_numpy_deg2rad(
```
Comment (fspyridakos) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:3070]: `fn_tree` and the test name should refer to the function as jax.numpy.degrees, as that is the function name in the jax.numpy frontend.

**Review by fspyridakos — CHANGES_REQUESTED** [review]
I pointed out another change you should make. Also my previous comments have not been adressed, please go over them again before requesting another review. Thanks!

**Comment by ndurumo254** [comment]
@fspyridakos . am getting challanges in reverting the changes. i have done git reset --hard-- several times but although the branch is back dated the changes are not reverted.

**Comment by fspyridakos** [comment]
First of all, the function you're trying to implement is fine so good job! Now as to reverting changes, you could remove unwanted files from the PR as described here: https://stackoverflow.com/questions/38743912/remove-a-file-from-a-git-pull-request.

Or you could just go to the Files changed, tab and manually reset changes by copy-pasting the original into your files. You could also do this on a whole-file scale by going to the files with changes and copy-pasting the whole original file into it (deleting your local file beforehand), then adding your wanted changes.

**Pull request closed without merging**